libs/GameSystems/CMakeLists.txt

```diff
@@ -13,6 +14,7 @@ target_include_directories(${TARGET}
     PUBLIC ${INC_ROOT}
     PUBLIC ${PROJECT_NAME}_ecs
     PUBLIC ${PROJECT_NAME}_gameComponents
+    PUBLIC ${PROJECT_NAME}_gameSharedResources
```

Comment:
You're not linking the library inside the Cmake, so why do you link her include path ?